src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -111,22 +126,23 @@ class DaemonPantsRunner:
   @classmethod
   def create(cls, sock, args, env, services, scheduler_service):
     maybe_shutdown_socket = MaybeShutdownSocket(sock)
+    exception = None
 
+    # TODO This can probably be moved to the try:except block in DaemonPantsRunner.run() function
```

Comment:
Linking to a ticket that explains why, or expanding the description here would be good. There is definitely dead code in here.

Reply:
Opened #8002, I hope it's enough for most people to pick up, but would definitely love feedback to make it more approachable.